code/ashley/django/todo_app_proj/templates/todo/index.html

```diff
@@ -0,0 +1,19 @@
+
+{% extends 'layout/base.html' %}
+
+{% block content %}
+     <h1>Todo List</h1>
+
+     <button href="{% url 'todo_app:add_item' %}">
```

Comment:
`button` does not allow the use of `href` should be an `a` tag or a bootstrap formatted `btn`